pvacseq/lib/main.py

```diff
@@ -93,21 +97,44 @@ def main(args_input = sys.argv[1:]):
 #            ).stdout.decode().split('\n')
 #        )
 
+    tmp_dir = tempfile.TemporaryDirectory()
+    tmp_split_fasta_prefix = os.path.join(tmp_dir.name, args.sample_name + "_" + str(args.peptide_sequence_length) + ".fa.split")
+
+    run(["split --lines=400 %s %s.split" % (fasta_file_path, tmp_split_fasta_prefix)], shell=True)
```

Comment:
If I'm reading the docs correctly (https://docs.python.org/3/library/subprocess.html#subprocess.run), it seems like we may either want to specify the `check=True` flag so it raises an exception on failure, or check the `returncode` value on the object that `run()` returns, either directly or using the `check_returncode` method.

Otherwise I believe there's a chance this could fail and then we'd proceed with the rest of the tool without knowing the split failed.

It also looks like maybe if we switch the value of `shell` to be `false` and provide a list of arguments instead of a single string, it will do escaping for us. I'm not sure if that's needed or not - could the filenames ever contain special characters, spaces, etc?